File: client/store/message.js
export const state = () => ({
  list: []
})

export const mutations = {

  addMessage(state,message){

    state.list.push({
      id:Math.random(),
      sender:message.sender,
      receiver:message.receiver,
      text:message.text
      ,
    })
  },
}

export const actions = {


}

File: server/src/main/java/app/models/Message.java
package main.java.app.models;

public class Message {

    private String text,sender,receiver;

    public Message() {
    }

    public Message(String sender,String receiver,String text) {
    	this.sender = sender;
        this.text = text;
        this.receiver = receiver;
    }

    public String getText() {
        return text;
    }
    
    public String getSender() {
    	return sender;
    }
    
    public String getReceiver() {
    	
    	return receiver;
    }

    public void setText(String text) {
        this.text = text;
    }
}